### PR TITLE
F51 96 featured products

### PR DESCRIPTION
**Pull request by amrarick26**
Directive for products where product.xp.Featured = true

### CODE REVIEW

File: src/app/common/directives/oc-featured-products/oc-featured-products.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                        }           
+                    ];
+                    element.html(
+                        "<br><h3 class='c-related-products__heading'>Featured Products</h3><hr>" +
+                        "<slick arrows='!application.isTouchDevice' responsive='responsive' infinite='false' slides-to-show='6' slides-to-scroll='1' class='slider multiple-items' ng-class='{\"has-arrows\": !application.isTouchDevice}'>" +
+                        "<div ng-repeat='product in featuredProducts'>" +
+                        "<div class='c-related-products__card-wrap' ng-include='\"common/directives/directives-product-card.html\"'></div>" +
+                        "</div>" +
+                        "</slick>"
+                    );
```
Comment (crhistianr) [inline src/app/common/directives/oc-featured-products/oc-featured-products.js]: Can we put this template in its own file like we did for that other directive?

Comment (amrarick26) [inline src/app/common/directives/oc-featured-products/oc-featured-products.js]: oc-related-products is like this too https://github.com/ordercloud-api/angular-buyer/blob/development/src/app/common/directives/oc-related-products/oc-related-products.js

With using a directive (slick) within a directive (this) it doesn't load the slick directive.

Comment (amrarick26) [inline src/app/common/directives/oc-featured-products/oc-featured-products.js]: @crhistianr this is done

**Comment by robertsoniv** [comment]
I will make a new PR with my branch

**Pull request closed without merging**